notebooks/setfit-absa-fiqa.ipynb

```diff
@@ -0,0 +1,7780 @@
+{
```

Comment:
<p># **Demo Code**</p><p>The following demonstrates the training of the SetFitABSA model and its evaluation over the <strong>**FiQA_SA**</strong> financial dataset.</p><p> </p><p>Specifically:</p><p> </p><p>* &nbsp; We sample <strong>**k=24**</strong> sentences from the training set using an arbitrary seed</p><p>* &nbsp; We trained the SetFitABSA model using only these <strong>**k**</strong> samples</p><p>* &nbsp; We evaluate the trained model over the entire test set which is exactly the same set that was used to evalute BloombergGPT model. The evaluated task is <strong>**SB2**</strong> - aspects are given and the model needs to predict their corresponding polarity</p><p> </p><p>With just 24 training sentences we get a weighted F1-score which is on par or better than the 75.07 score achieved by BloombergGPT (try it yourself!)</p><p>&lt;br&gt;</p><p>&lt;br&gt;</p><p>&lt;br&gt;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>